src/Http/Response.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+namespace Framework3\Http;
+
+use Framework3\Bag\Bag;
+use Framework3\Http\Session\Session;
+use Framework3\Http\Session\SessionInterface;
+
+class Response
+{
+    public const STATUS_OK =            ['code' => 200, 'name' => '200 OK'];
+    public const STATUS_NO_CONTENT =    ['code' => 204, 'name' => '204 No Content'];
+    public const STATUS_BAD_REQUEST =   ['code' => 400, 'name' => '400 Bad Request'];
+    public const STATUS_UNAUTHORIZED =  ['code' => 401, 'name' => '401 Unauthorized'];
+    public const STATUS_FORBIDDEN =     ['code' => 403, 'name' => '403 Forbidden'];
+    public const STATUS_NOT_FOUND =     ['code' => 404, 'name' => '404 Not Found'];
+
+    public const MIME_TEXT_PAIN =       "text/plain";
+    public const MIME_TEXT_HTML =       "text/html";
+    public const MIME_TEXT_JAVASCRIPT = "text/javascript";
+    public const MIME_TEXT_CALENDAR =   "text/calendar";
+    public const MIME_TEXT_CSV =        "text/csv";
+    public const MIME_APP_JSON =        "application/json";
+    public const MIME_APP_XML =         "application/xml";
+    public const MIME_APP_PDF =         "application/pdf";
+    public const MIME_APP_ZIP =         "application/zip";
+    public const CHARSET_UTF8 =         "UTF-8";
```

Comment:
Codacy found an issue: [Equals sign not aligned with surrounding assignments; expected 9 spaces but found 1 space](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6299285)